HomeworkTwo/Program.cs

```diff
@@ -1,16 +1,52 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Diagnostics.Eventing.Reader;
+using System.Runtime.Remoting.Channels;
+using System.Security.Cryptography.X509Certificates;
 
 namespace HomeworkTwo
 {
     class Program
     {
+        // TASK: Three parameters: Nominal of the transaction, Trade price, Transaction Type (Buy, sell), Sign (1 for buy, -1 for sell)
+        enum Transactiontype
```

Comment:
It is better to have new types in a separate file. Please move enum from Program.cs to TransactionType.cs. You can add a new file by right clicking on the project name and then selecting "add" > "class..."